etc/profile-m-z/signal-desktop.profile

```diff
@@ -32,12 +32,16 @@ caps.keep sys_admin,sys_chroot
 netfilter
 nodvd
 nogroups
+noroot
 notv
 nou2f
+novideo
+protocol unix,inet,inet6
 shell none
 
 disable-mnt
 private-dev
+private-etc machine-id,resolv.conf,fonts
```

Comment:
IMHO also these should be added.
TLS/SSL: `ca-certificates,ssl,pki,crypto-policies,nsswitch.conf`
Others: `alternatives,ld.so.cache,ld.so.conf,ld.so.conf.d,ld.so.preload`

Also sort it alphabetical.